### ISSUE TITLE
Number formatter on tooltips ignore axis configuration

### ISSUE THREAD

**Issue by VictorJuliani** [amcharts/amcharts4#249]
Hi!

It seems that both series and cursor tooltips use the wrong number formatter to display the data.
I created a repro to demonstrate it: https://codepen.io/victorjuliani/pen/rrXJEX?editors=0010

The chart has a global formatter to add a 'ºC' symbol and the Y Axis has a local formatter to add a '%' symbol.

What I believe to be expected is: 
1) x labels with ºC, x tooltips with ºC and series' formatter {categoryX} with ºC
2) y labels with %, y tooltips with % and series' formatter {valueY} with %

What is being shown:
1) x labels with ºC (ok), x tooltips with no formatter (**wrong**) and series' {categoryX} with ºC (ok)
2) y labels with % (ok), y tooltips with ºC (**wrong**) and series' {valueY} with ºC (**wrong**)

I have tested replacing the global formatter with a local one on X axis, but in this case, cursor tooltips aren't formatted at all.

Is the described behavior expected and I should use adapters + local formatters or is it really an issue?

**Comment by martynasma** [comment]
OK, so there's definitely some tangle of formatter inheritance going on, which we'll need to figure out.

At the very least axis tooltips should use the same formatter as the axis itself.

Not sure if it's possible to automatically figure out which formatter to use for the tooltip, but that's a different story.

I'll get back to you once we had a chance to figure this out.

**Comment by martynasma** [comment]
OK, so we now implemented proper inheritance of formatters for axes and series.

Tooltips on axis will use axis' formatter. Tooltips on series will use series' formatter.

Please note that this is not yet released. It will be at some point this week. Will let you know here.

Regrading formatting within tooltip. As I mentioned before, I think it would be too much of a fuzzy logic to try to format values according to the axis formatter. Tooltips will use series' formatters.

If you need to differentiate formats within single tooltip, you can use `formatDate`/`formatNumber` syntax:

```
series.columns.template.tooltipText = `[bold]{categoryX}[/]: {valueY.formatNumber('#.## %')}`;
```

https://www.amcharts.com/docs/v4/concepts/formatters/#Overriding_format_in_data_fields

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.47](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.47).

## [4.0.0-beta.47] - 2018-08-24

### Added
- New series type: `OHLCSeries`.
- `goHome()` method added to `MapChart`. Calling it will reset map to `homeZoomLevel` and `homeGeoPoint`.
- `homeGeoPoint` added to `MapChart`. Allows setting initial `geoPoint` at which map will be centered.
- `homeZoomLevel` added to `MapChart`. Allows setting initial `zoomLevel`.
- `maxPanOut` property added to `MapChart`, with default value `0.7`. It prevents map to be dragged out of the chart area.
- `noRisers` property added to `StepLineSeries`. It allows step line chart without vertical (or horizontal, depending on orientation) lines.
- Step line without risers now allows making Waterfall chart. Example added.
- `startLocation` and `endLocation` added to `StepLineSeries`, that allow controling locations at which step should start/end.
- `maxPrecision` added for `ValueAxis`. Use it to restrict Value axis to show value labels with less than desired precision (set 0 if you only want integers).
- Portugese (Portugal) translation added.

### Changed
- `Cursor` will now not move if there's some element obstructing the chart's plot area, e.g. a popup or some other element outside the chart.

### Fixed
- Series' and axes' tooltips will now correctly inherit formatters.
- Errors when hovering mouse cursor over open Modal.
- JSON: `children` property of `Container`-type objects was not being processed correctly.
- Popup/Modal now sets `aria-label` automatically if title is set.
- Removing bullet template from series or disposing it will remove/dispose all the bullets created from it.
- Tooltips were overlapping each other, fixed for the most part, but might still be some situations where they overlap.
- Changing data for `MapImageSeries` and `MapLineSeries` used to result runtime error.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
